frontend/src/Constants/ComponentText.ts

```diff
@@ -2,6 +2,11 @@ const ComponentText = {
   NEXT: 'Next',
   ADD: 'Add',
   CANCEL: 'Cancel',
+  ADD_FIELD: 'Add Field',
+  BACK: 'Back',
+  // For buttonSize arguments. Refer to ButtonSizeConstant.ts
+  ISDESKTOP: true,
+  ISMOBILE: false,
```

Comment:
I'd say this file is for text that goes in the components, can this go somewhere else?